src/messages/MOSDOp.h

```diff
@@ -30,8 +30,10 @@
  *
  */
 
-class OSD;
+class MOSDOpReply;
 
+namespace _mosdop {
+template<typename V>
```

Comment:
this isn't really constructed much outside the objecter - was there a particular reason to template this instead of changing it to small_vector here?

Reply:
Originally I just changed it, but this ended up either requiring me to copy into a regular std::vector or make more extensive modifications to the OSD.

Reply:
makes sense, I suspected that was the problem